### PR TITLE
Drop support for PostgreSQL 12

### PR DESCRIPTION
**Pull request by jedcunningham**
It reached EOL on 2024-11-14.

### CODE REVIEW

**Review by potiuk — APPROVED** [review]

File: dev/breeze/doc/images/output_testing_db-tests.svg [hunk 1]
```diff
@@ -0,0 +1,117 @@
+<svg class="rich-terminal" viewBox="0 0 1482 391.59999999999997" xmlns="http://www.w3.org/2000/svg">
```
Comment (potiuk) [inline dev/breeze/doc/images/output_testing_db-tests.svg]: Drop this file. It must have been added before rebase / latest changes with redesigning the test commands.

Comment (jedcunningham) [reply dev/breeze/doc/images/output_testing_db-tests.svg]: Ah good eye. Weird.

Comment (potiuk) [reply dev/breeze/doc/images/output_testing_db-tests.svg]: it's the pre-commit that spotted it - not me :).

BTW. And now you need to re-run pre-commit (`breeze static-check --only-my-changes`) as well as it needs to regenerate the images with PostgreSQL 12 removed :(

**Review by eladkal — APPROVED** [review]

**Comment by eladkal** [comment]
Do we need newsfragment for this one?

File: dev/breeze/doc/images/output_testing_db-tests.svg (hunk 1 above)
Comment (jedcunningham) [reply dev/breeze/doc/images/output_testing_db-tests.svg]: My breeze was in a pretty unhappy state somehow - it had removed 12 from a few but not all. But should be good now. Need to move `breeze down` higher on my troubleshooting list...

**Comment by jedcunningham** [comment]
> Do we need newsfragment for this one?

Good idea. We've been pretty inconsistent about this when dropping support for versions of things that hit EOL, but I think we should call it out explicitly.

**Review by eladkal — APPROVED** [review]

**Pull request merged**